tests/e2e/specs/in-game/countering.spec.js

```diff
@@ -283,12 +283,17 @@ describe('Countering One-Offs', () => {
     cy.get('#counter-dialog').should('be.visible').get('[data-cy=counter]').click();
     cy.get('#choose-two-dialog').should('be.visible').get('[data-counter-dialog-card=2-2]').click();
     cy.get('#waiting-for-opponent-counter-scrim').should('be.visible');
+    cy.get('#waiting-for-opponent-counter-scrim [data-overlay-one-off=1-0]').should('exist');
+    cy.get('#waiting-for-opponent-counter-scrim [data-overlay-counter=2-2]').should('exist');
```

Comment:
Interestingly, these have to be `'exist'` assertions rather than `'be.visible'` because the cards cover each other which cypress considers to mean not visible